### DISCUSSION TITLE
List-KR-Fast 요 부분은 혹시 언제

### DISCUSSION THREAD

**Discussion by swbaek81** [List-KR/community#6] [Q&A]
늘 빠른 피드백 감사드립니다
https://github.com/List-KR/List-KR-Fast
혹시 이 프로젝트는 일정이 대략적으로 라도 
어떻게 되는지 궁금해요

listkr 도 관리하셔야 하고 바쁘실거는 같지만요

**Comment by piquark6046** [comment]
하고 있긴 한데, 제가 너무 귀찮아서 확실하게 말씀을 못 드리겠습니다.
요청이 온 김에 더 집중해보겠습니다.
